### PR TITLE
Fix for CVE-2021-3121

### PR DESCRIPTION
**Pull request by satadrubiswas**
CVE https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-3121 is fixed in package gogo/protobuf more than year back. This pull request is to ensure that this repo is using the latest version of gogo/protobuf

- Upgrade github.com/gogo/protobuf to v1.3.2 https://github.com/gogo/protobuf/releases/tag/v1.3.2
- Upgrade github.com/golang/protobuf to v1.5.2

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,9 +4,8 @@ go 1.12
 
 require (
 	github.com/gogo/googleapis v0.0.0-20180223154316-0cd9801be74a
-	github.com/gogo/protobuf v1.2.0
-	github.com/golang/protobuf v1.2.0
-	golang.org/x/sync v0.0.0-20190227155943-e225da77a7e6 // indirect
+	github.com/gogo/protobuf v1.3.2
+	github.com/golang/protobuf v1.5.2
```
Comment (johanbrandhorst) [inline go.mod]: I don't think we can introduce this version, it breaks compatibility with <v1.3.x. Can you revert this version change?

**Review by johanbrandhorst — CHANGES_REQUESTED** [review]
Don't we actually need to regenerate the files as well, not just update the dependency?

**Comment by nabbar** [comment]
any news about this update ?
@johanbrandhorst could you make the update of code if necessary ?

**Review by johanbrandhorst — APPROVED** [review]

**Comment by sahilvv** [comment]
@johanbrandhorst can a new release be cut to be able to consume this change?

**Comment by johanbrandhorst** [comment]
Thanks for the reminder, I just cut `v1.1.1`.

**Pull request merged**